reana_client/utils.py

```diff
@@ -119,20 +119,24 @@ def load_reana_spec(filepath, skip_validation=False):
             _validate_reana_yaml(reana_yaml)
 
         kwargs = {}
-        if reana_yaml['workflow']['type'] == 'serial':
+        workflow_type = reana_yaml['workflow']['type']
+        if workflow_type == 'serial':
             kwargs['specification'] = reana_yaml['workflow'].\
                 get('specification')
             kwargs['parameters'] = \
                 reana_yaml.get('inputs', {}).get('parameters', {})
             kwargs['original'] = True
 
+        if 'options' in reana_yaml['workflow']:
```

Comment:
We should add this new field to the [schema](https://github.com/reanahub/reana-client/blob/master/reana_client/schemas/reana_analysis_schema.json).